.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,49 @@
+If you encounter a bug or you have a support question, please include the following information:
+
+### Subject of the issue
+Describe your issue here.
+
+### Your environment
+* Output of `haskell-language-server --probe-tools` or `haskell-language-server-wrapper --probe-tools`
```

Comment:
Maybe we should mention that this cli flag is not available in the latest release? Or wait till the next release?

Reply:
I think even after we release the next version, this template should mention the version range in which this command is available. If not, some users will try to run those command with outdated HLS, and then may think this issue template is outdated (or somewhat broken) and ignore the template.